### PR TITLE
Added exclusions for php7.4 packages

### PR DESCRIPTION
**Pull request by daviddjh**

### CODE REVIEW

**Comment by ramereth** [comment]
@daviddjh can you also add support for 7.4?

**Comment by daviddjh** [comment]
Yup! Doing that now

**Comment by daviddjh** [comment]
@ramereth Rake passes if that rubocop:disable line is removed, didn't know whether it should be.

**Comment by ramereth** [comment]
@daviddjh looks like we have a Cookstyle issue you should fix for ``test/cookbooks/php_test/recipes/phpinfo_apache.rb``. Please fix this on violet which will catch it.

```
Inspecting 38 files
.......................W..............

Offenses:

test/cookbooks/php_test/recipes/phpinfo_apache.rb:3:18: W: Lint/SendWithMixinArgument: Use include Apache2::Cookbook::Helpers instead of send(:include, Apache2::Cookbook::Helpers).
::Chef::Resource.send(:include, Apache2::Cookbook::Helpers)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

38 files inspected, 1 offense detected
```

**Comment by daviddjh** [comment]
will do

File: test/cookbooks/php_test/recipes/phpinfo_apache.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 major_version = node['platform_version'].to_i < 8 ? node['php']['version'].to_i : 7
 
-::Chef::Resource.send(:include, Apache2::Cookbook::Helpers)
+include Apache2::Cookbook::Helpers
```
Comment (ramereth) [inline test/cookbooks/php_test/recipes/phpinfo_apache.rb:3]: This actually needs to be changed to this:
```suggestion
::Chef::Resource.include Apache2::Cookbook::Helpers
```

**Comment by daviddjh** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v5.3.0.  Have a nice day!

**Pull request merged**